pkg/codegen/testing/test/sdk_driver.go

```diff
@@ -574,8 +585,8 @@ func TestSDKCodegen(t *testing.T, opts *SDKCodegenOptions) { // revive:disable-l
 			files, err := GeneratePackageFilesFromSchema(schemaPath, opts.GenPackage)
 			require.NoError(t, err)
 
-			if !RewriteFilesWhenPulumiAccept(t, dirPath, opts.Language, files) {
-				expectedFiles, err := LoadBaseline(dirPath, opts.Language)
+			if !RewriteFilesWhenPulumiAccept(t, e.CWD, opts.Language, files) {
```

Comment:
Doesn't this break the PULUMI_ACCEPT flow? Because we're going to rewrite the files in the temp dir?

Reply:
Yeah it does.  We probably need to re-export these files. 

The latest commit fixes that.